File: PubeoAPI/DTO/ParticipationSimpleDTO.cs
using System;
using System.ComponentModel.DataAnnotations;
using System.ComponentModel.DataAnnotations.Schema;

namespace PubeoAPI.DTO {
    public class ParticipationSimpleDTO {
        
        public Guid ParticipationId { get; set; }
        public Guid ParticulierId { get; set; }
    }
}

File: PubeoApp/app/src/main/java/com/example/pubeo/DTO/ParticipationSimpleDTO.java
package com.example.pubeo.DTO;

public class ParticipationSimpleDTO {

    private String participationId;
    private String particulierId;

    public ParticipationSimpleDTO(String participationId, String particulierId) {
        this.participationId = participationId;
        this.particulierId = particulierId;
    }

    public String getParticipationId() {
        return participationId;
    }

    public String getParticulierId() {
        return particulierId;
    }
}
